contracts/network/BancorNetwork.sol

```diff
@@ -665,10 +665,10 @@ contract BancorNetwork is IBancorNetwork, Upgradeable, ReentrancyGuardUpgradeabl
         CompletedWithdrawal memory completedRequest = _pendingWithdrawals.completeWithdrawal(contextId, provider, id);
 
         if (completedRequest.poolToken == _bntPoolToken) {
-            return _withdrawBNT(contextId, provider, completedRequest);
+            return _withdrawBNT(contextId, provider, _pendingWithdrawals, completedRequest);
```

Comment:
why pass a state var to an internal method?

Reply:
Since it's not immutable, I wanted to save the extra read. Too much?

Reply:
I must be missing something - where is the extra read?

Reply:
Two lines above, but it wasn't cached obviously

Reply:
Discussed and removed 👍